<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>700</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Bahnschrift Light</family>
    <pointsize>18</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #808080</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>374</x>
      <y>170</y>
      <width>321</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>120</y>
      <width>411</width>
      <height>461</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>341</width>
      <height>551</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #fff</string>
    </property>
    <item>
     <property name="text">
      <string>ПГНИУ
(Г. Пермь, ул. Генкеля, 8)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ПГНИУ
(Г. Пермь, ул. Генкеля, 7)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Дворец молодежи
(Петропавловская, 185)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Espresso Bar «ZAPRAVKA»
(ул. Монастырская, д.14)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Кофейня &quot;Karamely&quot;
(Пермь, Екатерининская, 122)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Первый корпус ПГПУ
(ул. Сибирская, 24)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Этно эко центр &quot;Нанук&quot;
(Елькина, 45)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>кинотеатр Премьер
(Пионерская, 17)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Магазин гаражных распродаж &quot;Полка Чудес&quot;
(ул. Героев Хасана, 28)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Поместье Алая Роза
(Красные Казармы 7)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>&quot;Толкучка&quot; комиссионный магазин
(Белинского 31, 2й этаж)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Выставочный центр Пермская Ярмарка
(бул. Гагарина, 65)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Уют Пекарня
(Юрша 100)</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>610</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #D8BFD8</string>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>791</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #D8BFD8</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:16pt; font-weight:600; color:#ffffff;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; color:#ffffff;&quot;&gt;АДРЕСА КНИЖНЫХ ПОЛОК&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>790</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
